cluster/ubuntu/util.sh

```diff
@@ -243,6 +242,7 @@ KUBE_APISERVER_OPTS="\
  --insecure-port=8080\
  --etcd-servers=http://127.0.0.1:4001\
  --logtostderr=true\
+ --storage-backend=etcd2\
```

Comment:
I think you are right, the default value should be used, but error occurs after install, so i changed the etcd's version, PTAL, thank you!